packages/ant-component-mapper/src/common/form-wrapper.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Form } from 'antd';
+
+const AntForm = ({ label, children, layout, component, invalid, isRequired, help }) => {
+  return (
+    <Form layout={layout} component={component}>
```

Comment:
You can try to remove the form here and put it in the FormWrapper in FormTemplate to make performance better

Reply:
The layout here decides if the label and item are horizontally or vertically aligned.  How can I pass the prop to FormWrapper from each component?

Reply:
Also on doing that, "required error" doesn't seem to work on Submit.

Reply:
You can pass `layout` props to FormTemplate, so everyone can decide that layout the form will use and I think it should be common for all fields. (`vertical` can be default)

If requried error does not work, please check if there is no additional component between `Form` and `Form.Item`